### ISSUE TITLE
how long it takes for main_lstM_etallica.py

### ISSUE THREAD

**Issue by SeekPoint** [keunwoochoi/LSTMetallica#7]
In my gtx1080, it takes 3 days and still going to

**Comment by keunwoochoi** [comment]
For how many epoches? That seems too long.

**Comment by SeekPoint** [comment]
I just run     python main_lstM_etallica.py

now it is still print:

23188/23188 [==============================] - 681s - loss: 0.0028  
Epoch 1/1
23188/23188 [==============================] - 669s - loss: 0.0022  
Epoch 1/1
23188/23188 [==============================] - 673s - loss: 0.0036  
Epoch 1/1
23188/23188 [==============================] - 662s - loss: 0.0036  
Epoch 1/1
23188/23188 [==============================] - 696s - loss: 0.0023  
Epoch 1/1
23188/23188 [==============================] - 682s - loss: 0.0028  
Epoch 1/1
 7552/23188 [========>.....................] - ETA: 441s - loss: 0.0038

**Comment by unsalted** [comment]
You set a while loop with a boolean value in front of it `batch_size_not_decided` so it never leaves the while loop, is that intentional @keunwoochoi ?
```python
    batch_size_not_decided: True
    while batch_size_not_decided:
         result = model.fit(X, y, batch_size=batch_size, nb_epoch=nb_epoch, callbacks=[checker, early_stop]) 
         loss_history = loss_history + result.history['loss']
```

**Comment by keunwoochoi** [comment]
I completely forgot about the lines and they seem strange ;)

**Comment by unsalted** [comment]
What is supposed to go there (if anything at all)?

**Comment by keunwoochoi** [comment]
Well, there should be a way to break the loop. I really don't understand why there isn't.

**Comment by unsalted** [comment]
@lovejasmine running the code on a 1080 with a while loop not exiting was like leaving the lights on with no one in the room for 3 days  ; ) - I've been there, fortunately not today.

**Comment by unsalted** [comment]
@keunwoochoi Do you have a branch or commit from when you ran it (correctly).  Because right now `batch_size_not_decided` is never set to `False` (or at all) in the code.

**Comment by keunwoochoi** [comment]
@unsalted I removed the lines. Turned out that I made a mistake while cleaning the code, the code is definitely unnecessary. Thanks for pointing out.

**Comment by unsalted** [comment]
Sure thing, thanks for the fix!
